config/initializers/alaveteli.rb

```diff
@@ -11,7 +11,7 @@
 load "util.rb"
 
 # Application version
-ALAVETELI_VERSION = '0.22.4.3'
+ALAVETELI_VERSION = '0.22.4.4'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.